nixos/modules/system/boot/systemd-unit-options.nix

```diff
@@ -233,6 +233,7 @@ in rec {
 
     path = mkOption {
       default = [];
+      type = with types; listOf package;
```

Comment:
```suggestion
      type = types.package;
```

Reply:
But why? It's a list of packages that gets mushed into a string via the `apply` function